spring-batch-core/src/test/java/org/springframework/batch/core/repository/support/SimpleJobRepositoryTests.java

```diff
@@ -16,10 +16,7 @@
 
 package org.springframework.batch.core.repository.support;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
We tend to avoid wildcard imports in new code (we basically follow the [same code style as SF](https://github.com/spring-projects/spring-batch/blob/master/CONTRIBUTING.md#code-style) which [forbids](https://github.com/spring-projects/spring-framework/wiki/Code-Style#import-statements) them), so could you please replace that with individual imports? I know we have a couple of those in the code base now, but we will fix them separately another time.

Reply:
Changed back to individual imports according to code style